tests/integration/routes/test_confirmation_email.py

```diff
@@ -255,10 +255,9 @@ def test_thank_you_missing_email(self):
 
         # Then I get an error message on the thank you page
         self.assertInUrl("/submitted/thank-you/")
-        self.assertInBody("There is a problem with this page")
         self.assertInBody("Enter an email address")
         self.assertEqualPageTitle(
-            "Error: Thank you for completing the census - Confirmation email test schema"
+            "We’ve received your answers - Confirmation email test schema"
```

Comment:
This test is kind of confusing now, nothing in it to suggest that having a missing email generates an error?

Reply:
I've changed all error message tests that I've touched so they use more specific CSS selector of error panel, I hope you like it.